.github/workflows/tests.yml

```diff
@@ -14,8 +14,12 @@ jobs:
       - uses: actions/checkout@v1
       - name: Select Xcode 11.2
         run: sudo xcode-select -s /Applications/Xcode_11.2.app
+      - name: Install carthage binaries
```

Comment:
can we please pull in the prebuild binaries via an external repo in the WORKSPACE?

Reply:
Mmm I do not fully follow. You mean create a git repo, push the binaries there and consume them using `http_file`?

Note that the safest is to build the binaries ourselves, because of the situation with swift ABI stability (prebuilt binaries built with a version of swift that does not match the one being used will result in an error)

Reply:
I dont think it makes sense to depend on carthage inside the tests for the rules -- it also makes it impossible to just run `bazel test ...` locally. If we're using carthage to download binaries, why can we not skip that and do the download via bazel?

Reply:
I see what you mean. The problem is that Carthage is not downloading the binaries, is building them. Solution I see is to somehow call Carthage from bazel, maybe with a genrule. What do you think?

Reply:
But I am not sure how easy that genrule can be created :D

Reply:
Hey @segiddins, I managed to create a genrule that builds the binaries with carthage and zips the build results:
```
    native.genrule(
        name ="carthage_genrule",
        srcs = glob(["Cartfile.resolved"]),
        outs = ["Carthage.zip"],
        cmd = """
        set -euo pipefail

        # Calculate absolut paths
        OUTPUT="$$(pwd)/$@"
        RESOLVED_CARTFILE="$$(pwd)/$<"
        CARTFILE_PARENT="$$(dirname "$${RESOLVED_CARTFILE}")"

        cd "$${CARTFILE_PARENT}"
        carthage bootstrap --no-use-binaries --platform iOS > /dev/null
        zip -r -X -q "$${OUTPUT}" "Carthage"
        rm -rf "Carthage"
        """,
        tags = tags + ["manual"] + ["no-sandbox"],
    )
```

The problem is that now I can not find a way of passing the `InputMask.framework/**/*` files that are zipped inside `Carthage.zip` to the `apple_static_framework_import_patched` rule:
```
apple_static_framework_import_patched(
    name = "InputMask",
    framework_imports = glob([":carthage/Build/iOS/InputMask.framework"]), <== How can I do this?
    visibility = ["//visibility:public"],
)
```

Any ideas?

Reply:
Okey, I managed to do it as you suggested :)